extensions/cloud/portability-cloud-google/src/main/java/org/datatransferproject/cloud/google/GoogleJobStore.java

```diff
@@ -333,4 +337,46 @@ private Key getDataKey(UUID jobId, String key) {
       throw new RuntimeException("Failed to deserialized entityKey: " + entityKey, t);
     }
   }
+
+  @Override
+  public void create(UUID jobId, String key, InputStream stream) throws IOException {
+    Preconditions.checkNotNull(jobId);
```

Comment:
can you make a helper method and refactor most of this? I think the only difference between this and the other create() is the objectMapper line